app/views/browse/_show.html.haml

```diff
@@ -71,6 +71,10 @@
           Used in 1 class.
         - else
           Used in #{material_count} classes.
+        - if material.respond_to?(:archived?) && material.archived?
+          (archived)
+        = link_to "(portal settings)", edit_polymorphic_url(material)
```

Comment:
I'm not sure but it seems like this 'portal settings' link might be shown to any user viewing this browse page.  That wouldn't be good because it won't work unless the user has permission.  I think there is a policy used to figure out if this link should be added to the search results, so that same policy could be used here.